docs/build-using-docker.md

```diff
@@ -0,0 +1,38 @@
+# Build Using Docker
+
+<!-- TOC depthFrom:2 -->
+
+- [Linux Build Instructions](#linux-build-instructions)
+- [Windows Build Instructions](#windows-build-instructions)
+
+<!-- /TOC -->
+
+To avoid installing and configuring all the tools such as the Oracle JDK locally on your machine you can use Docker to build the desktop variant and dedicated server of "JSettlers" in an isolated environment.
+
+## Linux Build Instructions
+
+Run the following commands on Linux based systems:
+
+```
+git clone https://github.com/subesokun/settlers-remake.git
```

Comment:
Why should users clone from your repo instead of the main settlers repo?

Reply:
Sorry, this is of course a misstake. I I'll change it.

Reply:
solved issue